mypy/fixup.py

```diff
@@ -286,6 +286,11 @@ def lookup_qualified_stnode(modules: Dict[str, MypyFile], name: str,
         if not rest:
             return stnode
         node = stnode.node
+        # In fine-grained mode, could be a cross-reference to a deleted module
+        if node is None:
```

Comment:
The comment for `SymbolTable.node` says that `node` is None for bound type variables, but here this is not what we are expecting, so apparently that comment is not quite right. I'm assuming that if this condition is true, we should have a cross reference. Maybe check/assert that `cross_ref` attribute is not None if that's the case? Also worth updating the comment for `SymbolTable.node` to document that the value can be None in this case as well (assuming I'm right in my reasoning here).